tap_listrak/schemas/message_clicks.json

```diff
@@ -1,41 +1,36 @@
 {
-  "type": "object",
-  "properties": {
-    "MsgID": {
-      "type": [
-        "integer"
-      ]
+    "type": "object",
+    "properties": {
+        "ContactID": {
```

Comment:
MsgID is a parent id and we are inserting it in the schema explicitly to map child record with the parent record. Therefore it should not be removed from the schema.